### DISCUSSION TITLE
Crash and error reporting

### DISCUSSION THREAD

**Discussion by MrMage** [TelemetryDeck/Viewer#82] [Ideas]
Are you planning to support crash reporting, and maybe even error reporting? The latter might otherwise be accomplishable by logging custom events for errors.

**Comment by winsmith** [comment]
Of course it's attractive to be the one-stop solution for all of analytics, error reporting, but it's not my main focus right now. 

There are pretty good tools like the Apple crash reporter and Sentry, and for now I'm going to spend my time making the analytics part REALLY good, before even thinking of competing against those. 

If you yourself want to log custom signals for errors, there's nothing stopping you (just make sure you don't put PII into the metadata), but it's probably not the ideal solution, UI wise.
